_posts/2025-05-13-developer-update.md

```diff
@@ -0,0 +1,133 @@
+---
+author: Jody Garnett
+layout: post
+title: GeoServer 2025 Q2 Developer Update
+date: 2025-05-13
+categories:
+- Behind The Scenes
+---
+
+The GeoServer team charging ahead with our [2025 roadmap plans]({% post_url 2025-01-13-roadmap %}).
+
+{% include 2025-sponsors.html %}
+
+## CITE Certification
+
+A great deal of progress has been made on CITE Certification with the most recent [GeoServer 2.27.0 Release]({% post_url 2025-04-03-geoserver-2-27-0-released %}) passing tests! This is great for interoperability and project stability as the CITE tests act as external "blackbox" testing framework and verifies that GeoServer is operating as intended. 
+
+We are presently determining how to pay for certification:
+
+* The Open Source Geospatial Foundation has negotiated a reduced rate of $150 annual cost per standard certified.
+* We are prioritizing tests where we can act as a "reference implementation" resulting in no annual cost for OSGeo.
+* For Web Feature Service we pass tests for WFS 2.0, WFS 1.1, and WFS 1.0 which would add up to $450. It may be worth while only being certified for the latest WFS 2.0 to reduce the costs to $150.
+* There are also now CITE Tests for output formats. This would allow us WFS and WPS output to be certified on individual formats like GeoPackage and GeoTIFF.
+
+While GeoServer presently "implements" these standards, our sponsorship level is not sufficient to allow us to feel comfortable paying an annual costs.
+
+Full certification amounts to $900 a year, while certifying only the latest services amounts to $450 a year.
+
+| OGC Standard |  | Full Certification |  | Latest Services  |
+| ------------ | -----: | ------ | -----: | ------ | 
+| *Services* | | |
+| OGC API - Features | $150 | Certified | $150 | Certified |
+| WCS 2.0.1 | $0 | Reference | $0 | Reference |
+| WCS 1.1.1 | $0 | Reference | $0 | Reference |
+| WCS 1.0 | $0 | Reference | $0 | Reference |
+| WFS 2.0 | $150 | Certified | $150 | Certified |
+| WFS 1.1.0 | $150 | Certified | $0 | Implements |
+| WFS 1.0.0 | $0 | Reference | $0 | Reference |
+| WMS 1.3.0 | $150 | Certified | $150 | Certified |
+| WMS 1.1.1 | $150 | Certified | $0 | Implements |
+| WMTS 1.0.0 | $0 | Reference | $0 | Reference |
+| *Data formats and encodings* | | |
+| GeoTIFF 1.1 | $150 | Certified |$0 | Implements |
+| GeoPackage 1.2 | $150 | Certified | $0 | Implements |
+| | |
```

Comment:
This extra blank row is to provide some space from the text below.
I cannot figure out the CSS styling for tables on our website, grid lines are not showing up. Even messing around in browser inspector and adjusting and I cannot get borders shown

<img width="757" alt="image" src="https://github.com/user-attachments/assets/7f419e77-b333-4b1c-acd7-d971617e8da7" />